test/performance/lxd-profile.yaml

```diff
@@ -0,0 +1,105 @@
+description: "LXD profile for Canonical Kubernetes"
```

Comment:
if you need this profile.  can you pull it from https://github.com/canonical/k8s-snap/blob/main/tests/integration/lxd-profile.yaml

Reply:
Okay, since I don't expect to configure it in the future, I'm removing the configuration option- we can always add it back later if we need it.